docs/Datagrid.md

```diff
@@ -881,25 +881,26 @@ const PostList = () => (
 );
 ```
 
-The inspector uses the field `source` (or `label` when it's a string) to display the column name. If you use non-field children (e.g. action buttons), then it's your responsibility to wrap them in a component with a `label` prop, that will be used by the inspector:
+The inspector uses the field `source` (or `label` when it's a string) to display the column name. If you use non-field children (e.g. action buttons), then it's your responsibility to wrap them in a component with a `label` prop, that will be used by the inspector. You can use a [`<WrapperField>`](./WrapperField.md) for that purpose:
 
 ```tsx
-const FieldWrapper = ({ children, label }) => children;
 const PostList = () => (
     <List>
         <DatagridConfigurable>
             <TextField source="id" />
             <TextField source="title" />
             <TextField source="author" />
             <TextField source="year" />
-            <FieldWrapper label="Actions">
+            <WrapperField label="Actions">
                 <EditButton />
-            </FieldWrapper>
+            </WrapperField>
         </DatagridConfigurable>
     </List>
 );
 ```
 
+**Tip:** You may need to clear your local storage to reflect the changes, as react-admin saves the computed column names in the Store. For the same reason, you users may need to log out and in again to see the changes. Alternatively, you can leverage [Store Invalidation](./Store.md#store-invalidation) to do it automatically.
```

Comment:
```suggestion
**Tip:** You may need to clear your local storage to reflect the changes, as react-admin saves the computed column names in the Store. For the same reason, your users may need to log out and in again to see the changes. Alternatively, you can leverage [Store Invalidation](./Store.md#store-invalidation) to do it automatically.
```

Reply:
woops, I'll fix it on `master` then

Reply:
Done https://github.com/marmelab/react-admin/commit/00053dc73dfb30bd3391af16a44b156776f480ed